generators/typescript-v2/ast/src/ast/Method.ts

```diff
@@ -0,0 +1,51 @@
+import { CodeBlock } from "./CodeBlock";
+import { Comment } from "./Comment";
+import { Parameter } from "./Parameter";
+import { Type } from "./Type";
+import { Func } from "./Func";
+import { Reference } from "./Reference";
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+
+export declare namespace Method {
+    type Args = {
+        reference: Reference;
+        name: string;
+        parameters: Parameter[];
+        return_: Type[];
+        body?: CodeBlock;
+        docs?: string;
+    };
+}
+
+export class Method extends Func {
```

Comment:
Similar to above, I don't think we need this node to start. If there are still TODOs, it's probably best to just leave it out for now. Plus, it looks like the snapshots are actually generating top-level functions instead of methods?

```typescript
function withBaseURL(baseUrl: string): string {
    console.log("foo");
    return baseUrl + "foo";
}
```

Reply:
Yeah, we can probably leave a lot of it stubbed.